x-pack/test_serverless/api_integration/config.base.ts

```diff
@@ -32,6 +36,10 @@ export function createTestConfig(options: CreateTestConfigOptions) {
         serverArgs: [
           ...svlSharedConfig.get('kbnTestServer.serverArgs'),
           `--serverless=${options.serverlessProject}`,
+          // custom native roles are enabled only for search and security projects
+          ...(options.serverlessProject !== 'oblt'
+            ? ['--xpack.security.roleManagementEnabled=true']
+            : []),
```

Comment:
Just leaving myself a note that this (an equivalent in functional and agnostic test configs) can be removed once https://github.com/elastic/kibana/pull/195584 is merged.